src/UserSettingsStore.js

```diff
@@ -25,50 +25,7 @@ import SdkConfig from './SdkConfig';
  * TODO: Make things use this. This is all WIP - see UserSettings.js for usage.
  */
 
-const FEATURES = [
```

Comment:
Is it worth deleting UserSettingsStore entirely to force everything over to the new API?

Reply:
This still does a fair amount of user stuff; although it has been gutted of all settings-related things. It could probably be moved to UserSettings, although that component is quite bulky.